### PR TITLE
Switch to GOV.UK design system

### PR DESCRIPTION
**Pull request by ushkarev**
MTP apps used to use a combination of [govuk-template](https://github.com/alphagov/govuk_template), [govuk-elements](https://github.com/alphagov/govuk_elements) and [govuk-frontend-toolkit](https://github.com/alphagov/govuk_frontend_toolkit). These are deprecated in favour of the more modern, flexible and accessible [govuk-frontend](https://github.com/alphagov/govuk-frontend).

Switching to the new framework involves changing ~pretty much~ every template, style and javascript module because govuk-frontend uses completely different patterns. It was decided that, during this upgrade, we should minimise our use of customised MTP-specific components where possible in favour of those provided by GDS.

Depends on [mtp-common#405](https://github.com/ministryofjustice/money-to-prisoners-common/pull/405)

[MTP-1308](https://dsdmoj.atlassian.net/browse/MTP-1308)

### CODE REVIEW

**Review by xoen — DISMISSED** [review]
The nature and size of this PR makes it tricky to review but I had a look and it looks good to me 👍

File: mtp_cashbook/apps/disbursements/views.py [hunk 1]
```diff
@@ -776,34 +775,68 @@ def form_valid(self, form):
...
+            dict(link, active=index == index_active)
+            for index, link in enumerate(cls.sub_navigation_links)
+        ]
+
+
+class ProcessOverview(DisbursementHelpView):
     url_name = 'process-overview'
 
+    def get_context_data(self, **kwargs):
+        kwargs['sub_navigation_links'] = self.make_sub_navigation_links(0)
```
[the remark this answers is not in the document]
Comment (ushkarev) [reply mtp_cashbook/apps/disbursements/views.py]: 🙆‍♂️

**Review by xoen — APPROVED** [review]
LGTM 👍

**Review by xoen — APPROVED** [review]

File: mtp_cashbook/assets-src/components/disbursements/index.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    this.initPendingList();
+  },
+
+  initAmount: function () {
+    // update NOMIS balances without page reload
+    $('.mtp-accounts-table tfoot a').click(function (e) {
+      e.preventDefault();
+      var $button = $(this);
+      var $dataBoxes = $('.mtp-accounts-table__amount');
+      var errorMessage = django.gettext('Please try again later');
```
Comment (Staberinde) [inline mtp_cashbook/assets-src/components/disbursements/index.js:18]: Similar comment about django javascript being in scope to above

**Review by Staberinde — COMMENTED** [review]
First pass: Just a few comments

File: mtp_cashbook/apps/disbursements/forms.py [hunk 3]
```diff
@@ -151,7 +151,6 @@ def to_python(self, value):
 class AmountForm(DisbursementForm):
     amount = AmountField(
         label=_('Amount to send'),
-        help_text=_('For example, 10.00'),
```
[the remark this answers is not in the document]
Comment (ushkarev) [reply mtp_cashbook/apps/disbursements/forms.py:154]: this was dead code, it wasn't displayed

File: mtp_cashbook/assets-src/components/cashbook/index.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  },
+
+  initSelectionCount: function () {
+    // displays a running count of selected credits
+    $('.mtp-input--selection-count').each(function () {
+      var $countContainer = $(this);
+      $('.mtp-input--counted').on('change', function () {
+        var itemCount = $('.mtp-input--counted:checked').length;
+        if (itemCount > 0) {
+          var message = django.ngettext(
```
[the remark this answers is not in the document]
Comment (ushkarev) [reply mtp_cashbook/assets-src/components/cashbook/index.js]: so i "fixed" it but it's really not very elegant… not sure what's best to do :/

**Review by xoen — APPROVED** [review]
👍

**Pull request merged**